client/index.tsx

```diff
@@ -1,10 +1,7 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import App from "./App";
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import App from './App';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById("app")
-);
\ No newline at end of file
+const rootNode: HTMLElement = document.getElementById('app')!;
+const root = createRoot(rootNode);
+root.render(<App />);
```

Comment:
There was an error being thrown in the console for using ReactDOM.render instead of createRoot. This fixes that error.